Lava-Puddles-Map/map.conf

```
@@ -0,0 +1,15 @@
+team.1.pos = (-98,1,102)
+r = 115
+initial_stuff = default:pick_steel
+team.2 = blue
+rotation = z
+name = Lava Puddles
+author = smugler5
+h = 140
+team.1.color = red
+team.2.color = blue
+team.2.pos = (101,1,-102)
+team.1 = red
+licence = CC BY-SA
+treasures = shooter:shotgun, default:pick_steel, default:pick_steel, default:dirt 70, default:stone 60

```

Incorrect, this should be `shooter_guns:shooter`.
```suggestion
treasures = shooter_guns:shotgun, default:pick_steel, default:pick_steel, default:dirt 70, default:stone 60
```